_component/modal/_notes.md

```diff
@@ -1 +1,14 @@
-The modal dialog component is a standard and accessible JavaScript plugin that was converted from a jQuery plugin by Scott Ohara, you can [view the plugin on Github](https://github.com/scottaohara/accessible_modal_window). This component is keyboard accessible, reusable, and properly traps focus while the dialog is open. It also exposes a small API via a callback function you can hook into if you need to extend the functionality. No jQuery is required.
+We highly recommend the following:
+
+[a11y-dialog](http://edenspiekermann.github.io/a11y-dialog/) is a lightweight (1.3Kb) yet flexible script to create accessible dialog windows.
+
+✔︎ No dependencies  
```

Comment:
@colorful-tones @garand This is rendering out as a paragraph, should be an unordered list

Reply:
I originally copy/pasted from `a11y-dialog` [`README.md`](https://github.com/edenspiekermann/a11y-dialog/blob/master/README.md)

I now added bullets.